operations/mimir-mixin/recording_rules.libsonnet

```diff
@@ -479,7 +479,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
         rules: [
           {
             // cortex_ingester_ingested_samples_total is per user, in this rule we want to see the sum per cluster/namespace/instance
-            record: 'cluster_namespace_%s:cortex_ingester_ingested_samples_total:rate1m' % $._config.per_instance_label,
+            record: '%s_namespace_%s:cortex_ingester_ingested_samples_total:rate1m' % [$._config.clusterLabel, $._config.per_instance_label],
```

Comment:
The actual aggregation is dynamic and based on `alert_aggregation_labels`. It's only the recording rule name that reference `cluster_namespace` hardcoded. Instead of using `$._config.clusterLabel` here, we should build the recording rule prefix based on `alert_aggregation_labels`.  See this related issue: https://github.com/grafana/mimir/issues/1632